src/hooks/check_undeclared_software.py

```
@@ -268,34 +272,51 @@ def main(
+    process_status = console.status("[bold yellow] Running SCANOSS scan...\n")
     process_status.start()
 
+    logging.debug(f"Executing command: {sanitize_scan_command(scanoss_scan_cmd)}")
+
     try:
-        scan_result = run_subcommand(scanoss_scan_cmd)
+        scan_result = run_subcommand(scanoss_scan_cmd, debug=debug)
     except subprocess.CalledProcessError as e:
         logging.error(
             f"Error running scanoss command with return code {e.returncode}: {e.stderr}"
         )
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)